app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -1,104 +1,87 @@
 package com.rentminder
 
-import androidx.compose.foundation.layout.Arrangement
```

Comment:
Removed all imports that were not being used. These cluttered up the imports if you need to review them later. Additionally, they bloated the line numbers.

Reply:
While organizing imports is a good idea, this can be performed automatically by the IDE.  No need to do it manually and create a new branch.